sw/device/lib/dif/dif_dma.h

```diff
@@ -0,0 +1,414 @@
+// Copyright lowRISC contributors (OpenTitan project).
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+
+#ifndef OPENTITAN_SW_DEVICE_LIB_DIF_DIF_DMA_H_
+#define OPENTITAN_SW_DEVICE_LIB_DIF_DIF_DMA_H_
+
+/**
+ * @file
+ * @brief <a href="/hw/ip/dma/doc/">DMA Controller</a> Device Interface
+ * Functions
+ */
+
+#include <stdint.h>
+
+#include "dma_regs.h"  // Generated.
+#include "sw/device/lib/dif/autogen/dif_dma_autogen.h"
+
+#ifdef __cplusplus
+extern "C" {
+#endif  // __cplusplus
+
+// Target Address space that the source address pointer refers to.
+typedef enum dif_dma_address_space_id {
+  /*OpenTitan 32 bit internal bus.*/
```

Comment:
nit: the spacing of the comment seems wrong, you can ignore it if it's too annyoing to change. It also happens on the next item.